decidim/decidim
# Fix initiative print link, margin, and organization logo

#### :tophat: What? Why?

This PR removes a legacy print page that we have in admin. This should have been removed when #5736 was implemented, but it was left. I've removed the action and linked to the frontend.  

In the process of fixing it, I found a couple more bugs that this PR also solves:

1. If the initiative has a logo uploaded then you'll have an exception here: 

> 
>   Failure/Error: <%= image_tag current_organization.logo.url, title: current_organization.name %>
>   
>   ActionView::Template::Error:
>    bad URI(is not URI?): nil

I think it was because it wasn't migrated during the change to ActiveStorage on #7598. I've also added a regression test for this. 

2. There was a string left without i18n translation. I've added it.
3. There was a missing margin in the page. I've added it. (see screenshots) 

#### :pushpin: Related Issues

- Fixes #9028 

#### Testing

0.  Upload the logo to the organization 
1. Go to an initiative in "Technical validation" state.
2. Click in the Print link 
3. See that you have the organization logo
4. See that you have a margin in this page
5. (After the i18n gets filled in Crowdin, you can check that the "Members.." string is translated)

Regarding the print initiative in admin:

1. Sign in as admin 
2. Go to the initiatives' page in the admin panel 
3. Click in the print button
4. See that it isn't in Catalan and doesn't have the wording for the city of Barcelona 

### :camera: Screenshots

#### Before

![image](https://user-images.githubusercontent.com/717367/163171597-62d2b92f-2659-4436-a6d1-d5f8018b9688.png)


#### After

![image](https://user-images.githubusercontent.com/717367/163171621-17c511ad-db84-4b25-9247-bb378df68ffb.png)


:hearts: Thank you!
